routing/routing_session.hpp

```diff
@@ -194,9 +194,10 @@ class RoutingSession : public traffic::TrafficObserver, public traffic::TrafficC
 
   /// RemoveRoute removes m_route and resets route attributes (m_state, m_lastDistance, m_moveAwayCounter).
   void RemoveRoute();
-  void RemoveRouteImpl();
   void RebuildRouteOnTrafficUpdate();
 
+  void ResetImpl();
```

Comment:
Am I right that this method must be called only when mutex is locked? If so, please, add a comment here.